src/slider/slider.jsx

```diff
@@ -231,11 +244,22 @@ export default class Slider extends Component {
 
         return (
             <ConfigProvider rtl={false}>
-                <div role="toolbar" dir="ltr" className={clazz} style={style} {...obj.pickOthers({...Slider.propTypes, ...InnerSlider.propTypes}, sliderProps)}>
-                    <InnerSlider ref={InnerSlider => (this.innerSlider = InnerSlider)} {...sliderProps} />
+                <div
+                    role="list"
+                    dir="ltr"
```

Comment:
not related to this change, but why is the `dir` fixed to `ltr`?

Reply:
There are a lot of calculations in Slider, `dir` fixed to `ltr` to simplify the calculations when rtl.